litsearch/getInternalData.py

```diff
@@ -0,0 +1,160 @@
+import pandas as pd
+import requests
+import json
+import base64
+
+
+# Issues Helper Functions
+def getIssuesFromAPI():
+    """ Gets all the issues and pull-requests (GH treats PRs like issues in the api)
+    Needs to use pagination because of 100 per-request api limit
+    """
+    issues = []
+    pageNumber = 1
+    numberOfIssuesReturned = 1
+    while numberOfIssuesReturned != 0:
+        issuesResponse = requests.get(
+            "https://api.github.com/repos/greenelab/covid19-review/issues?state=all&per_page=50&page=" +
+            str(pageNumber))
+        issues_page = json.loads(issuesResponse.text)
+        issues = issues + issues_page
+        numberOfIssuesReturned = len(issues_page)
+        pageNumber += 1
+    return issues
+
+
+def getCitationFromIssue(issue):
```

Comment:
Most of our issues have DOIs.  If we wanted to be more general, we could use the Manubot package API to convert the citation (in any form: URL, PMID, DOI, etc.) to a CSL Item and then extract the DOI from the CSL if it exists.  That will be slower and not provide much benefit if the vast majority of issues have DOIs.

There are some corner cases like #109 that have atypical citations.

Reply:
This could be cool but I agree with you that it wouldn't provide much benefit at this point.

We can try to list the DOI in the issues and if this becomes problematic (i.e. the number of new paper issues being submitted by others increases and manually going into issues and putting the DOI becomes too much) we can revisit this.